wallet-new/src/Cardano/Wallet/Kernel/DB/Spec/Update.hs

```diff
@@ -28,7 +36,7 @@ import           Cardano.Wallet.Kernel.DB.Util.AcidState
 -- | Errors thrown by 'newPending'
 data NewPendingFailed =
     -- | Some inputs are not in the wallet utxo
-    NewPendingInputUnavailable (InDb (Core.TxIn))
+    NewPendingInputsUnavailable (Set (Core.TxIn))
```

Comment:
Why did you get rid of the `InDb` here? Now the `SafeCopy` instance for `NewPendingFailed` can potentially use a different serialization of these transactions inputs than we use everywhere else.